src/main/java/nl/utwente/presto/tezos/TezosSplit.java

```diff
@@ -40,6 +42,26 @@ public static TezosSplit forBlock(long blockId) {
         return new TezosSplit(TezosTable.BLOCK, Type.BLOCK, blockId);
     }
 
+    /**
+     * Create new splits for a range of blocks
+     * @param blockIdStart lower bound block height (ID)
+     * @param blockIdEnd upper bound block height (ID)
+     * @return new splits
+     */
+    public static List<ConnectorSplit> forBlockRange(long blockIdStart, long blockIdEnd) {
```

Comment:
Returns List<ConnectorSplit> instead of List<TezosSplit> the latter is harder to work with in TezosSplitManager